cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go

```diff
@@ -0,0 +1,226 @@
+package dpdk
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strings"
+
+	sriovv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	sriovcluster "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/cluster"
+	sriovnamespaces "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/namespaces"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/networks"
+	utilNodes "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/nodes"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/performanceprofile"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/pods"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/nodes"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+)
+
+var _ = Describe("[sriov] NUMA node alignment", Ordered, func() {
+
+	BeforeEach(func() {
+		if discovery.Enabled() {
+			Skip("Discovery mode not supported")
+		}
+
+		isSNO, err := utilNodes.IsSingleNodeCluster()
+		Expect(err).ToNot(HaveOccurred())
+		if isSNO {
+			Skip("Single Node openshift not yet supported")
+		}
```

Comment:
None of the below tests make changes to these conditions. Should these be moved into BeforeAll?

Reply:
You are right! This comes from a bad habit we developed while working with the custom version of [`BeforeAll()`](https://github.com/openshift-kni/cnf-features-deploy/blob/dad3bccf61b7b33ed1319a69f8810745157af7a4/cnf-tests/testsuites/pkg/execute/ginkgo.go#L7), where skipping in the passed function would skip only the first test in the node.

BTW Fixed